src/test/lib/optimizer/strategy/predicate_reordering_rule_test.cpp

```diff
@@ -149,11 +149,11 @@ TEST_F(PredicateReorderingTest, PredicatesAsRightInput) {
      *
      *             _______Cross________
      *            /                    \
-     *  Predicate_0(a > 80)     Predicate_2(a > 90)
+     *  Predicate(a > 80)     Predicate(a > 90)
      *           |                     |
-     *  Predicate_1(a > 60)     Predicate_3(a > 50)
+     *  Predicate(a > 60)     Predicate(a > 50)
      *           |                     |
-     *        Table_0           Predicate_4(a > 30)
+     *        Table_0           Predicate(a > 30)
```

Comment:
Formatting

Reply:
In what way? What's the issue?

Reply:
"Predicate" is not aligned with the Predicates above anymore. Really minor though. ;)